tests/testTranslationRecovery.cpp

```diff
@@ -69,10 +71,14 @@ TEST(TranslationRecovery, BAL) {
 
   // Check result for first two translations, determined by prior
   EXPECT(assert_equal(Point3(0, 0, 0), result.at<Point3>(0)));
-  EXPECT(assert_equal(Point3(2 * w_aZb.point3()), result.at<Point3>(1)));
+  EXPECT(assert_equal(Point3(2 * w_aZb_stored.point3()), result.at<Point3>(1)));
```

Comment:
why this multiplication by 2?

Reply:
Because scale = 2.0. The result is expected to be the point at a distance 2 in the direction along w_aZb